app/views/chapters/_chapter.html.erb

```diff
@@ -8,8 +8,8 @@
   <% end %>
   <% if chapter.user_has_creator_invite?(current_user) %>
     <p class="notice">
-      <%= ts("You've been invited to become a co-creator of this chapter. To accept or reject the invitation, visit your %{creator_invitations_page}.",
-             creator_invitations_page: link_to(ts("Creator Invitations page"), user_creatorships_path(current_user))).html_safe %>
+      <%= ts("You've been invited to become a co-creator of this chapter. To accept or reject the request, visit your %{creator_requests_page}.",
```

Comment:
It looks like the Jira issue expects the text to still say "To accept or reject the _invitation_," but I think this is probably okay. (At some point, we might want to think of different phrasing than "You've been invited," though.)